src/NUnitFramework/framework/Attributes/TestFixtureSourceAttribute.cs

```diff
@@ -198,8 +198,8 @@ public IEnumerable<ITestFixtureData> GetParametersFor(Type sourceType)
             if (SourceName == null)
                 return Reflect.Construct(sourceType) as IEnumerable;
 
-            MemberInfo[] members = sourceType.GetMember(SourceName,
-                    BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Static | BindingFlags.Instance | BindingFlags.FlattenHierarchy);
```

Comment:
Thanks for catching this. It's a little awkward for our purposes that `FlattenHierarchy` doesn't handle private members